e2e/terraform/gke/oebs-staging.tf

```diff
@@ -0,0 +1,29 @@
+
+provider "google" {
+  credentials = "${file("${var.credentials}")}"
+  project     = "${var.project}"
+  region      = "${var.region}"
+}
+
+resource "google_container_cluster" "primary" {
+  name = "${var.clustername}"
+  zone = "${var.region}"
+  initial_node_count = "${var.nodecount}"
+
+  master_auth {
+    username = "admin"
+    password = "oebsstaging@1234"
```

Comment:
PASSWORD?

Reply:
This is a placeholder @AmitKumarDas . However, will change it to an ENV based on review